extensions/ql-vscode/src/interface.ts

```diff
@@ -357,7 +357,7 @@ export class InterfaceManager extends DisposableObject {
           t: 'ExtensionParsed',
           pageNumber: 0,
           numPages: numPagesOfResultSet(resultSet),
-          resultSet,
+          resultSet: { t: 'RawResultSet', ...resultSet },
```

Comment:
Has the type `RawResultSet` changed? In the current main branch, it looks like there is no `t` property.

Reply:
To accomodate sarif results, I changed the type of the `.resultSet` field of `ExtensionParsedResultSets` from being always only a `RawResultSet` to being a `ResultSet` (which is either a `RawResultSet` or a `SarifResultSet`, tagged appropriately)